__tests__/cmds/versions.test.js

```diff
@@ -180,29 +172,24 @@ describe('rdme versions*', () => {
         .basicAuth({ user: key })
         .reply(200, { version });
 
-      return deleteVersion.run({ key, version }).catch(err => {
-        expect(err.message).toContain('The version you specified');
-        mockRequest.done();
-      });
+      await expect(deleteVersion.run({ key, version })).rejects.toThrow('The version you specified');
```

Comment:
Wanna also check for the full `APIError` object here as well? (see this comment)